tck/src/main/java/org/eclipse/microprofile/openapi/tck/AirlinesAppTest.java

```diff
@@ -1021,4 +1021,44 @@ public void testExceptionMappers(String type) {
         vr.body("paths.'/user/{id}'.get.responses.'404'.content.'application/json'.schema", notNullValue());
     }
 
+    @RunAsClient
+    @Test(dataProvider = "formatProvider")
+    public void testAdditionalPropertiesFalse(String type) {
+        ValidatableResponse vr = callEndpoint(type);
+
+        String responseSchema =
+                dereference(vr, "paths.'/bookings/{id}'.get.responses.'200'", "content.'application/json'.schema");
+        vr.body(responseSchema, notNullValue());
+        String ccSchema = dereference(vr, responseSchema, "properties.creditCard");
+
+        vr.body(ccSchema + ".additionalProperties", equalTo(false));
+
+    }
+
+    @RunAsClient
+    @Test(dataProvider = "formatProvider")
+    public void testAdditionalPropertiesTrue(String type) {
+        ValidatableResponse vr = callEndpoint(type);
+
+        String responseSchema =
+                dereference(vr, "paths.'/bookings/{id}'.get.responses.'200'", "content.'application/json'.schema");
+
+        vr.body(responseSchema, notNullValue());
+
+        String airlineSchema = dereference(vr, responseSchema, "properties.returningFlight", "properties.airline");
+        vr.body(airlineSchema + ".additionalProperties", equalTo(true));
```

Comment:
The OpenAPI spec states that `true` is the default value for `additionalProperties`, so I think it should be valid to omit it.

Reply:
Updated test to check for both valid options for `true`. However, I am on the fence about this one. Although `true` and `additionalProperties` being absent are equivalent from the standpoint of OpenAPI semantics, it might make more sense to honor the application's explicit configuration of the `true` value and require it in the output.

Reply:
I don't really mind either way. I added the comment because it's important that the TCK doesn't test anything that isn't specified, so if there are two valid ways of doing something the TCK must allow either.

However, looking back at the javadoc for `True`, it does look like you do explicitly require that `true` is included in the model, so I think you are ok to check for `true` in the output.
> Marker class for use in additionalProperties to indicate that the corresponding schema's additionalPropertiesBoolean value is to be set to boolean true

Reply:
This check has been restored.